### ISSUE TITLE
Fix DockerImage class unit tests

### ISSUE THREAD

**Issue by PrajwalM2212** [vmware/tern#405]
**Problem Statement**
The import statements on test files are wrong and need to be rearranged.

**Describe the Proposal**
Describe the change you would like to see in the project.
https://github.com/vmware/tern/blob/master/tests/test_class_docker_image.py
uses 
```python
from classes.docker_image import DockerImage
from utils.container import docker_command
from utils.container import pull
from utils.container import check_image
```
but 
https://github.com/vmware/tern/commit/8092d0fde3059de9115ae485fd7d09cf67642e22 along with a number of PR's had made refactors a lot of code making these import statements outdated. 
This has to be fixed for the tests to run

**Steps to Implement Proposal**
Refactor the import statements in the test files.

**Comment by nishakm** [comment]
In order to make this a "real" unit test, we need a test Docker image. But we do not have a project Docker repository. We will wait until tern has its own Dockerhub repo.

**Comment by nishakm** [comment]
So we have a repo now `https://hub.docker.com/r/vmware/tern`. Next thing we need is a dummy test image. We can create one this way:
1. Create a simple directory structure:
```
foo
     |_ file1 <-owned by user
     |_ file2
     |_ bar
              |_ file1 <-owned by root
              |_ file2
```
2. Run fs_hash.sh on this directory, get the checksum and save it somewhere
3. Tar up this directory
4. Do a `docker load < testdir.tar`
5. Do a `docker push vmware/tern/testimage:<date>`
6. Do a `docker save` of this image, untar the contents and save all of the checksums, manifest and config information for the test.
7. Write the test that checks to see if all the content is the same for this image.

Note that we should be pulling this image by digest if we are to run tests against it.

**Comment by PrajwalM2212** [comment]
I am working on it.

**Comment by nishakm** [comment]
Fixed with PR #536
